stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/TextFieldConfig.kt

```diff
@@ -7,7 +7,7 @@ import androidx.compose.ui.text.input.VisualTransformation
 import kotlinx.coroutines.flow.StateFlow
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP_PREFIX)
-sealed interface TextFieldConfig {
```

Comment:
is this expected?

Reply:
Yes. sealed classes don't allow implementations in different modules.